api/meetings/serializers.py

```diff
@@ -38,6 +38,8 @@ class MeetingSerializer(JSONAPISerializer):
     active = ser.BooleanField(read_only=True)
     submission_1_email = ser.SerializerMethodField()
     submission_2_email = ser.SerializerMethodField()
+    accept_type_one = ser.BooleanField(source='poster', read_only=True)
+    accept_type_two = ser.BooleanField(source='talk', read_only=True)
```

Comment:
🎉 thanks for updating this for me @adlius! Do you mind adding a couple of lines in an existing test for this? There's a test testing what is serialized on a meetings detail view.

Reply:
Done.

Reply:
It's bothering me that we're using digits in the email fields (e.g. `submission_1_email`) and written out numbers in the accept fields (e.g. `accept_type_one`). IMHO, we should pick one.

Reply:
good eye! I just used submission_1_email to mirror what was in the db in the Conference.field_names dict - which also uses digits, but I generally prefer things spelled out.

Reply:
I took the liberty to change that.

Reply:
Ok, this may be getting nit picky, but I like more descriptive fields and for booleans to start with `is`, `has`, `can`, etc.:
```
type_one_submission_email
type_two_submission_email
is_accepting_type_one
is_accepting_type_two
```

Reply:
Changed.